### PR TITLE
Enable new header feature flag in production

### PR DESCRIPTION
**Pull request by zackkrida**
Enable the new header feature flag in production.

### CODE REVIEW

File: feat/feature-flags.json [hunk 1]
```diff
@@ -11,10 +11,10 @@
     "new_header": {
       "status": {
         "staging": "enabled",
-        "production": "disabled"
+        "production": "enabled"
       },
       "description": "Toggle the new header, including recent searches.",
-      "defaultState": "off"
+      "defaultState": "on"
```
Comment (obulat) [inline feat/feature-flags.json:17]:
```suggestion
```
I think you only need `defaultState` when the flag is switchable.

**Review by obulat — APPROVED** [review]
🚀

File: feat/feature-flags.json (hunk 1 above)
Comment (dhruvkb) [reply feat/feature-flags.json:17]: Yes, `defaultState` has no effect when status is 'enabled'.

**Review by dhruvkb — APPROVED** [review]
Approving as this works. The changes are non-blocking.

File: feat/feature-flags.json [hunk 2]
```diff
@@ -11,10 +11,9 @@
     "new_header": {
       "status": {
         "staging": "enabled",
-        "production": "disabled"
+        "production": "enabled"
       },
       "description": "Toggle the new header, including recent searches.",
```
Comment (zackkrida) [inline feat/feature-flags.json:16]:
```suggestion
      "description": "Toggle the new header, including recent searches."
```

**Pull request merged**